cli/devlog.py

```
@@ -71,5 +71,90 @@ def record(story_id: str, action: str, date: datetime | None):
+    
+    # Success path: complete the bounded recording state
+    try:
+        state.complete_bounded_recording(date, story_id, duration, assume_utc=True)
+        click.echo(f"[OK] Bounded recording completed for {story_id} ({duration}s)")
+    except Exception as e:
+        click.echo(f"[ERR] Failed to complete bounded recording for story {story_id}: {e}")
+        # Clean up recording state on failure
+        try:
+            state.end_recording(date, story_id, assume_utc=True)
+            click.echo(f"[INFO] Recording state cleaned up for {story_id}")
+        except Exception as cleanup_error:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)